driver_test.go

```diff
@@ -12,23 +12,27 @@ func TestParseURI(t *testing.T) {
 	}{
 		{
 			"databricks://token:supersecret@example.cloud.databricks.com/sql/1.0/endpoints/12346a5b5b0e123a",
-			Options{Host: "example.cloud.databricks.com", Port: "443", Token: "supersecret", HTTPPath: "/sql/1.0/endpoints/12346a5b5b0e123a", MaxRows: 10000, Timeout: 0, LogOut: ioutil.Discard},
+			Options{Host: "example.cloud.databricks.com", Port: "443", Token: "supersecret", HTTPPath: "/sql/1.0/endpoints/12346a5b5b0e123a", ClientId: "go-dbsql", Version: "0.9.0", MaxRows: 10000, Timeout: 0, LogOut: ioutil.Discard},
```

Comment:
if in future we release a newer version of the SDK, let's say 0.10.0, do we have to update all these tests?

Reply:
Yes, in `DefaultOptions` we would have to change `Version: "0.10.0"` and then update these tests to reflect the new default. 
Disadvantage: slightly cumbersome for developers
Advantage: version changes should have a bit of friction to validate default settings